device/esp_tinyusb/usb_descriptors.c

```diff
@@ -62,6 +62,30 @@ const tusb_desc_device_t descriptor_dev_kconfig = {
     .bNumConfigurations = 0x01
 };
 
+#if (TUD_OPT_HIGH_SPEED)
+const tusb_desc_device_qualifier_t descriptor_qualifier_kconfig = {
```

Comment:
@roma-jam @tore-espressif Given that these are private, could we also rename the `kconfig` descriptors to default? I get that some of their values are set via Kconfig. But feature wise, they act as the default descriptors if none are provided by the user.

```suggestion
const tusb_desc_device_qualifier_t descriptor_qualifier_default = {
```

Likewise with the other `..._kconfig` descriptors in this header as well.

Reply:
I don't mind, I like `_default` more than `_kconfig`. 
Probably, for Device Descriptor `_kconfig` makes sense, because we can configure the VID/PID with KConfig.
But anyway it is the default descriptor which is used when user doesn't provide any other.

Reply:
@tore-espressif may I briefly ask you about your opinion? 
Rename `descriptor_name_kconfig` -> `descriptor_name_default`?

Reply:
I agree with renaming to `_default`